### PR TITLE
540 update highlight features document

### LINKED ISSUE #540: Update highlight features document
**Is your feature request related to a problem? Please describe.**
As we are closing to MONAI v0.2 release, need to update the highlight features document to include new features in v0.2.

### PR DESCRIPTION
**Pull request by Nic-Ma**
Fixes #540 .

### Description
This PR added v0.2 architecture and modules chart to the documents and update the highlight features document.

### Status
**Ready**

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or new feature that would cause existing functionality to change)
- [ ] New tests added to cover the changes
- [x] Docstrings/Documentation updated

### CODE REVIEW

File: docs/source/highlights_v0.1.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
-# Modules for public alpha
+# Modules in v0.1(public alpha)
```
Comment (wyli) [inline docs/source/highlights_v0.1.md:1]: the version number should be v0.1.0, we need a white space before `(`

File: docs/source/highlights_v0.2.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# Modules in v0.2
```
Comment (wyli) [inline docs/source/highlights_v0.2.md:1]: v0.2 -> v0.2.0

File: docs/source/highlights_v0.1.md (hunk 1 above)
Comment (Nic-Ma) [reply docs/source/highlights_v0.1.md]: updated in the latest commit.
Thanks.

File: docs/source/highlights_v0.2.md (hunk 2 above)
Comment (Nic-Ma) [reply docs/source/highlights_v0.2.md]: updated in the latest commit.
Thanks.

**Comment by Nic-Ma** [comment]
Hi @wyli ,

Thanks for your review.
I updated according to your comments.
Could you please help review again?
Thanks.

**Comment by wyli** [comment]
since we haven't tagged v0.2.0 yet, shall we hold this PR and merge just before tagging?

**Comment by Nic-Ma** [comment]
> since we haven't tagged v0.2.0 yet, shall we hold this PR and merge just before tagging?

Hi @wyli ,

I already updated to have all the highlight features for v0.2.0, could you please help review it?
Thanks.

**Pull request merged**